10_functions/lowercase.js

```diff
@@ -0,0 +1,6 @@
+var upperstr = "BEN BARBER IS SUCH A GOOD TEACHER";
+
+function lowercase(){
+    return upperstr.toLowerCase();
```

Comment:
`upperstr` is a variable outside of the function. Instead of using `upperstr` here, you need to add a parameter to the function, and use the parameter name instead of `upperstr`.